projects/plugins/jetpack/class.jetpack.php

```diff
@@ -1676,22 +1675,22 @@ public static function show_development_mode_notice() {
 
 			$notice .= ' ' . self::development_mode_trigger_text();
 
-			echo '<div class="updated" style="border-color: #f0821e;"><p>' . $notice . '</p></div>';
+			echo '<div class="updated" style="border-color: #f0821e;"><p>' . $notice . '</p></div>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- All provided text.
```

Comment:
For my own future reference, do we have any guidelines for choosing between ignoring the OutputNotEscaped and using some variety of `wp_kses()`?